lighthouse-core/test/audits/lazy-third-party-test.js

```diff
@@ -0,0 +1,131 @@
+
+/* eslint-env jest */
+describe('Lazy load third party resources', () => {
+  it('correctly identifies a lazy loadable third party resource', async () => {
+    const artifacts = {
+      devtoolsLogs: {defaultPass: networkRecordsToDevtoolsLog([
+        {
+          url: 'https://example.com',
+          startTime: 0,
+          endTime: 50,
+          transferSize: 2000,
+        },
+        {
+          url: 'https://widget.intercom.io/widget/tx2p130c',
+          startTime: 100,
+          endTime: 150,
+          transferSize: 4000,
+        },
+        {
+          url: 'https://js.intercomcdn.com/frame-modern.bb95039c.js',
+          startTime: 200,
+          endTime: 250,
+          transferSize: 8000,
+        },
+      ])},
+      traces: {defaultPass: createTestTrace({timeOrigin: 0, traceEnd: 2000})},
+      URL: {finalUrl: 'https://example.com'},
+    };
+
+    const settings = {throttlingMethod: 'simulate', throttling: {cpuSlowdownMultiplier: 4}};
+    const results = await LazyThirdParty.audit(artifacts, {computedCache: new Map(), settings});
+
+    expect(results.score).toBe(0);
+    expect(results.displayValue).toBeDisplayString('1 facade alternative available');
+    expect(results.details.items).toEqual([
+      {
+        productName: 'Intercom Widget',
+        facade: {
+          type: 'link',
+          text: 'React Live Chat Loader',
+          url: 'https://github.com/calibreapp/react-live-chat-loader',
+        },
+        transferSize: 12000,
+        blockingTime: 0,
+        subItems: {type: 'subitems', items: [
+          {
+            url: 'https://js.intercomcdn.com/frame-modern.bb95039c.js',
+            mainThreadTime: 0,
+            blockingTime: 0,
+            transferSize: 8000,
+            startTime: 200,
+            endTime: 250,
+          },
+          {
+            url: 'https://widget.intercom.io/widget/tx2p130c',
+            mainThreadTime: 0,
+            blockingTime: 0,
+            transferSize: 4000,
+            startTime: 100,
+            endTime: 150,
+          },
+        ]},
+      },
+    ]);
+  });
+
+  it('does not report first party resources', async () => {
+    const artifacts = {
+      devtoolsLogs: {defaultPass: networkRecordsToDevtoolsLog([
+        {url: 'https://youtube.com'},
+        {url: 'https://www.youtube.com/embed/tgbNymZ7vqY'},
+      ])},
+      traces: {defaultPass: createTestTrace({timeOrigin: 0, traceEnd: 2000})},
+      URL: {finalUrl: 'https://youtube.com'},
+    };
+
+    const settings = {throttlingMethod: 'simulate', throttling: {cpuSlowdownMultiplier: 4}};
+    const results = await LazyThirdParty.audit(artifacts, {computedCache: new Map(), settings});
+
+    expect(results).toEqual({
+      score: 1,
+      notApplicable: true,
+    });
+  });
+
+  it('only reports resources which can be lazy loaded', async () => {
+    const artifacts = {
+      devtoolsLogs: {defaultPass: pwaDevtoolsLog},
```

Comment:
worth adding a comment why this devtoolsLog/trace might trigger the audit otherwise? I assume it has third party requests, but they don't have facades?